extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js

```diff
@@ -0,0 +1,235 @@
+/*
+ * VisualEditor user interface WikiaInfoboxInsertDialog class.
+ */
+
+/**
+ * Dialog for inserting portable infobox templates.
+ *
+ * @class
+ * @extends ve.ui.FragmentDialog
+ *
+ * @constructor
+ * @param {Object} [config] Configuration options
+ */
+ve.ui.WikiaInfoboxInsertDialog = function VeUiWikiaInfoboxInsertDialog( config ) {
+	// Parent constructor
+	ve.ui.WikiaInfoboxInsertDialog.super.call( this, config );
+
+	// Properties
+	this.surface = null;
+};
+
+/* Inheritance */
+
+OO.inheritClass( ve.ui.WikiaInfoboxInsertDialog, ve.ui.FragmentDialog );
+
+/* Static Properties */
+
+ve.ui.WikiaInfoboxInsertDialog.static.name = 'wikiaInfoboxInsert';
+
+ve.ui.WikiaInfoboxInsertDialog.static.title = OO.ui.deferMsg( 'wikia-visualeditor-dialog-infobox-insert-title' );
+
+ve.ui.WikiaInfoboxInsertDialog.static.size = 'small';
+
+/* Methods */
+
+/**
+ * @inheritdoc
+ */
+ve.ui.WikiaInfoboxInsertDialog.prototype.getSetupProcess = function ( data ) {
+	return ve.ui.WikiaInfoboxInsertDialog.super.prototype.getSetupProcess.call( this, data )
+		.next( function () {
+			this.surface = data.surface;
+		}, this );
+};
+
+/**
+ * @inheritdoc
+ */
+ve.ui.WikiaInfoboxInsertDialog.prototype.getBodyHeight = function () {
+	return 600;
```

Comment:
is there a reason for using different values - `400` or `600` and what will happen if browser window is smaller than 600px?